src/Routes/GSheetImport.ts

```diff
@@ -86,8 +86,14 @@ export const upload = async (
   })
 
   try {
-    await updateDatabase(rows)
-    res.send(200)
+    updateDatabase(rows)
+    res
+      .status(200)
+      .send(
+        `Updating ${
+          rows.length
+        } records in the database, please wait a few minutes!`
+      )
```

Comment:
I think it's dangerous [as discussed] to just ignore failures calling to the database. Looking at `updateDatabase`, it seems to just iterate through collections doing an upsert for each, which should _not_ be unreliable. Is it possible the `getPriceGuidance` call is the one that might be buggy or erroring out?

Reply:
I hear you and have added some try/catch and logging around `getPriceGuidance` - if that IS what's causing the issue we'll at least be notified of it in the logs.